main.go

```diff
@@ -128,11 +127,6 @@ func Main() int {
 
 	mux := http.NewServeMux()
 	mux.Handle("/metrics", promhttp.InstrumentMetricHandler(reg, promhttp.HandlerFor(reg, promhttp.HandlerOpts{})))
-	mux.Handle("/debug/pprof/", http.HandlerFunc(pprof.Index))
```

Comment:
just use the stdlib pprof library, the tectonic operators import was probably caused by auto-imports accidentally